src/mesh_analysis/waku_message_log_analyzer.py

```diff
@@ -84,7 +89,7 @@ def _get_affected_node_log(self, data_file: str) -> Result[Path, str]:
         pod_log = reader.read()
 
         log_lines = [inner_list[0] for inner_list in pod_log[0]]
-        log_name_path = self._folder_path / f"{data_file.split('.')[0]}.log"
+        log_name_path = self._dump_analysis_dir / f"{data_file.split('.')[0]}.log"
```

Comment:
Same thing here. Hadn't you changed the two dir attributes for the `self._folder_path`?

Reply:
Yes, and then I realized it was dump. And split it again. One for reading data and another one for dumping results. That makes much more sense...